template-files/android/AndroidManifest.xml

```diff
@@ -320,15 +320,23 @@
       </intent-filter>
     </service>
     <meta-data
-      android:name="com.google.firebase.messaging.default_notification_icon"
+      android:name="expo.modules.notifications.default_notification_icon"
       android:resource="@drawable/shell_notification_icon" />
     <meta-data
-      android:name="com.google.firebase.messaging.default_notification_color"
-      android:resource="@color/colorAccent" />
+      android:name="expo.modules.notifications.default_notification_color"
+      android:resource="@color/notification_icon_color" />
     <service
       android:name=".fcm.FcmRegistrationIntentService"
       android:exported="false">
     </service>
+    <!-- Remove the following 2 when we drop support for Legacy Notifications -->
+    <meta-data
+      android:name="com.google.firebase.messaging.default_notification_icon"
+      android:resource="@drawable/shell_notification_icon" />
+    <meta-data
+      android:name="com.google.firebase.messaging.default_notification_color"
+      android:resource="@color/colorAccent" />
```

Comment:
Would we want to adjust the color here to `notification_icon_color` too?

Frankly speaking I don't think these values ever get used in Expo client — they're only used for Firebase **notification** messages which, as we know, shouldn't occur there as: developers can't send messages to Expo clients and we don't.

These values may however be used in standalone apps where developers are able to send Firebase notification messages to devices. Maybe we **should** use the same color here as for Expo notifications?

Reply:
> These values may however be used in standalone apps where developers are able to send Firebase notification messages to devices

I hadn't considered that use case, so yeah I'll change this to `notification_icon_color` and also change the comment:
```
<!-- Remove the following 2 when we drop support for Legacy Notifications -->
```
so that instead we have:
```
    <!-- Applied to Firebase data messages -->
    <meta-data
      android:name="expo.modules.notifications.default_notification_icon"
      android:resource="@drawable/shell_notification_icon" />
    <meta-data
      android:name="expo.modules.notifications.default_notification_color"
      android:resource="@color/notification_icon_color" />

    <!-- Applied to Firebase notification messages -->
    <meta-data
      android:name="com.google.firebase.messaging.default_notification_icon"
      android:resource="@drawable/shell_notification_icon" />
    <meta-data
      android:name="com.google.firebase.messaging.default_notification_color"
      android:resource="@color/notification_icon_color" />
```

which hopefully is clearer to anyone looking at this in the future